commands/run/run.go

```diff
@@ -101,3 +90,11 @@ func runAccordingE2E() error {
 
 	return nil
 }
+
+func cleanup0() {
```

Comment:
The function name is confusing.

Reply:
> The function name is confusing.

Because `cleanup` is already a package name, or do you prefer something like `doCleanup`?

Reply:
> That would be better than `cleanup0` :)

Done